lib/utils.ts

```diff
@@ -273,3 +314,145 @@ export function functionNameToDisplay(name: string): string {
 
   return result;
 }
+
+export function jsonSplitter(
+  json: any,
+  path: (string | number)[] = []
+): Chunk[] {
+  /*
+  Breaks down JSON into individual chunks of data. Each "Chunk" ia defined by its path
+  (i.e. where it is positioned in the json). And the data itself.
+  E.g. The chunk for the field 'b' in the json {a: {b: 1}} => [{path: ["a", "b"], data: 1}
+  */
```

Comment:
I think `/** ... **/` is the official way to do docstrings (my IDE shows this as a comment rather than as a docstring)

Also typo in `ia`